docs/lll.rst

```diff
@@ -0,0 +1,20 @@
+###
+LLL
+###
+
+.. _lll:
+
+LLL is a low-level language using s-expressions for the EVM.
+
+The Solidity repository contains an LLL compiler, which shares the assembler subsystem with Solidity.
+However, apart from maintaining that it still compiles, no other improvements are made to it.
+
+It is not built unless specifically requested:
+```bash
+$ cmake -DLLL=ON ..
```

Comment:
Looking at the CI build artifact, it seems like there are no linebreaks here (https://51302-40892817-gh.circle-artifacts.com/0/docs-html/lll.html).

Reply:
Fixed.